packages/example-nestjs/src/app.controller.ts

```
@@ -0,0 +1,31 @@
+
+@Controller()
+export class AppController {
+  private readonly logger = new Logger(AppController.name);
+  private readonly items = new Map<string, UserDto["data"]>();
+  private id = 0;
+
+  @TsRestHandler(contract)
+  async handler() {
+    return tsRestHandler(contract, {
+      create: async ({ body }) => {
+        const item = { id: String((this.id += 1)), ...body.data };
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)